misc/userscripts/qute-lastpass

```diff
@@ -80,7 +80,8 @@ def qute_command(command):
         fifo.flush()
 
 def pass_(domain, encoding):
-    args = ['lpass', 'show', '-x', '-j', '-G', '.*{:s}.*'.format(domain)]
+    domain = domain.replace('.', '\\.')
```

Comment:
That seems odd, as e.g. a `b` in the domain will now result in a (literal) `\b` in the regex, which (as can be seen below) has a special meaning. Why not use Python's `re.escape` instead?

Reply:
I think that replace will not treat the first parameter as a regex -- and other regex meta characters won't appear in domain names anyway -- but sure, it's better to use `re.escape`.

Reply:
Ah, right, I mixed this up with the escaping in `fake_key_raw`.

Reply:
Sounds good. I'm primarily a golang/rust developer so I am not familiar with these libraries. I appreciate your suggestions and help.